### ISSUE TITLE
NameError: uninitialized constant Errno::ENOMEDIUM

### ISSUE THREAD

**Issue by junaruga** [jruby/jruby#5257]
### Environment

Provide at least:
* JRuby version (`jruby -v`) and command line (flags, JRUBY_OPTS, etc)

```
version: jruby 9.1.16.0
JRUBY_OPTS="--server -J-Xms512m -J-Xmx1024m"
```

* Operating system and platform (e.g. `uname -a`)

```
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.5 LTS
Release:	14.04
Codename:	trusty
```

Other relevant info you may wish to add:
* Installed or activated gems: mongo
* Application/framework version (e.g. Rails, Sinatra): Nothing.
* Environment variables

### Expected Behavior

* Describe your expectation of how JRuby should behave, perhaps by showing how CRuby/MRI behaves.
* Provide an executable Ruby script or a link to an example repository.

`Errno::ENOMEDIUM::Errno` object exists.

### Actual Behavior

* Describe or show the actual behavior.
* Provide text or screen capture showing the behavior.
About below test case,
https://github.com/mongodb/mongo-ruby-driver/blob/master/spec/mongo/socket_spec.rb#L21
> raise SystemCallError.new('Test error', Errno::ENOMEDIUM::Errno)

I got `NameError: uninitialized constant Errno::ENOMEDIUM`
https://api.travis-ci.org/v3/job/406430973/log.txt

I did grep on HEAD of master branch. Then I could not find the implementation.

```
$ grep -r 'Errno::ENOMEDIUM' *
```

**Comment by junaruga** [comment]
I compared the result of `Errno.constants` between Ruby 2.5.1 and JRuby 9.1.16.0 on my local environment (Fedora 28).

Ruby 2.5.1: https://gist.github.com/junaruga/9f455234c81cff7580b0ffd65085fb8d
JRuby 9.1.16.0: https://gist.github.com/junaruga/cbd781907d7c7b282f90d36b5cdd0ce7

Some of the Errno codes are not included in the JRuby.

**Comment by junaruga** [comment]
```
$ diff ruby_2.5.1_errno_constants_sort.txt jruby_9.1.16_errno_constants_sort.txt 
5d4
< :EADV
9,10d7
< :EAUTH
< :EBADE
12d8
< :EBADFD
14,18d9
< :EBADR
< :EBADRPC
< :EBADRQC
< :EBADSLT
< :EBFONT
20,21d10
< :ECANCELED
< :ECAPMODE
23,24d11
< :ECHRNG
< :ECOMM
29d15
< :EDEADLOCK
32,33d17
< :EDOOFUS
< :EDOTDOT
38d21
< :EFTYPE
41d23
< :EHWPOISON
48d29
< :EIPSEC
51,64d31
< :EISNAM
< :EKEYEXPIRED
< :EKEYREJECTED
< :EKEYREVOKED
< :EL2HLT
< :EL2NSYNC
< :EL3HLT
< :EL3RST
< :ELIBACC
< :ELIBBAD
< :ELIBEXEC
< :ELIBMAX
< :ELIBSCN
< :ELNRNG
66d32
< :EMEDIUMTYPE
72,73d37
< :ENAVAIL
< :ENEEDAUTH
78,79d41
< :ENOANO
< :ENOATTR
81d42
< :ENOCSI
86d46
< :ENOKEY
89d48
< :ENOMEDIUM
92,93d50
< :ENONET
< :ENOPKG
100d56
< :ENOTCAPABLE
104,105d59
< :ENOTNAM
< :ENOTRECOVERABLE
107d60
< :ENOTSUP
109d61
< :ENOTUNIQ
113d64
< :EOWNERDEAD
117,120d67
< :EPROCLIM
< :EPROCUNAVAIL
< :EPROGMISMATCH
< :EPROGUNAVAIL
125d71
< :EREMCHG
127,129d72
< :EREMOTEIO
< :ERESTART
< :ERFKILL
131d73
< :ERPCMISMATCH
136d77
< :ESRMNT
138d78
< :ESTRPIPE
143,144d82
< :EUCLEAN
< :EUNATCH
148,149d85
< :EXFULL
< :NOERROR
```

**Comment by junaruga** [comment]
@headius Thanks for the information! As I usually do not use Java now, I would like to delegate someone for the help.

**Comment by ahorek** [comment]
@headius I fixed this and some other missing parts https://github.com/jnr/jnr-constants/pulls

should I regenerate them too? If so, let me know.

**Comment by junaruga** [comment]
@ahorek thank you for the working! :smiley:

**Comment by headius** [comment]
I believe this was fixed by jnr/jnr-constants#46 which seems to be the last time we regenerated the Linux constants. That would have been included in JRuby 9.1.14.0 at least.
